Pod/Classes/XExtensionItem.m

```diff
@@ -76,7 +105,7 @@ - (void)addEntriesToUserInfo:(id <XExtensionItemDictionarySerializing>)dictionar
     self.userInfo = ({
         NSMutableDictionary *mutableUserInfo = [[NSMutableDictionary alloc] initWithDictionary:self.userInfo];
         [mutableUserInfo addEntriesFromDictionary:dictionarySerializable.dictionaryRepresentation];
-        mutableUserInfo;
+        [mutableUserInfo copy];
```

Comment:
The `userInfo` property has a copy attribute on it, so calling copy here is not appropriate.